app/gen-server/migration/1663851423064-UserUUID.ts

```diff
@@ -1,6 +1,6 @@
-import {User} from 'app/gen-server/entity/User';
-import {makeId} from 'app/server/lib/idUtils';
+
 import {MigrationInterface, QueryRunner, TableColumn} from "typeorm";
+import {addRefToUserList} from "../sqlUtils";
```

Comment:
Please expand filenames in imports to `app/...` (helps in grist-electron, grist-static - or rather, it is hard to explain to people why relative paths will sometimes work and sometimes not there)